parsing/comments.ml

```diff
@@ -0,0 +1,167 @@
+(***********************************************************************)
+(*                                                                     *)
+(*                                OCaml                                *)
+(*                                                                     *)
+(*            Xavier Leroy, projet Cristal, INRIA Rocquencourt         *)
+(*                                                                     *)
+(*  Copyright 1996 Institut National de Recherche en Informatique et   *)
```

Comment:
I forgot about those. I've fixed the headers of comments.ml and comments.mli in a new commit.